samples/drivers/audio/speaker/boards/nucleo_f767zi.conf

```diff
@@ -0,0 +1 @@
+CONFIG_LVGL=n
```

Comment:
this really shouldn't be needed? LVGL is typically not enabled by default.

Reply:
Yes, you are right, I removed it.